docs/source/format/DissociatedIPC.rst

```diff
@@ -0,0 +1,335 @@
+  the body of this message. Because it is opaque, it can be anything the server wants
+  to use. The URI and identifier do not need to be given to the client via Flight RPC,
+  but could come across from any transport or protocol desired.
+
+Upon receiving a ``<want_data>`` request, the server *should* respond by sending a stream
+of messages consisting of the following:
+
+* A 5-byte prefix
+  - The first byte of the message indicates the type of message, currently there are only
+    two allowed message types (more types may get added in the future):
+    0) End of Stream
+    1) Flatbuffer IPC Metadata Message
+  - the next 4-bytes are a little-endian, 32-bit integer indicating the sequence number of
+    the message. The first message in the stream (**MUST** always be a schema message) **MUST**
+    have a sequence number of ``0``. Each subsequent message **MUST** increment the number by 
+    ``1``.
+* The full Flatbuffer bytes of an Arrow IPC header
+
+After sending the last metadata message, the server **MUST** indicate the end of the stream
+by sending a message consisting of **exactly** 5 bytes:
+
+* The first byte is ``0``, indicating an **End of Stream** message
+* The last 4 bytes are the sequence number (4-byte, unsigned integer in little-endian byte order)
+
+Data Stream Sequence
+''''''''''''''''''''
+
+If a single server is handling both the data and metadata streams, then the data messages
+**should** begin being sent to the client in parallel with the metadata messages. Otherwise,
+as with the metadata sequence, the standing state of the server is to wait for a **tagged**
+message with the ``<want_data>`` tag value, whose body indicates the dataset / data stream
+to send to the client.
+
+For each IPC message in the stream of data, a **tagged** message **MUST** be sent on the data
+stream if that message has a body (i.e. a Record Batch or Dictionary message). The 
+:term:`tag <Tag>` for each message should be structured as follows:
+
+* The *least significant* 4-bytes (bits 0 - 31) of the tag should be the 32-bit, little-endian sequence 
+  number of the message.
+* The *most significant* byte (bits 56 - 63) of the tag indicates the message body **type** as an 8-bit
+  unsigned integer. Currently only two message types are specified, but more can be added as
+  needed to expand the protocol:
+  0) The body contains the raw body buffer bytes as a packed buffer (i.e. the standard IPC
+     format body bytes)
+  1) The body contains a series of unsigned, little-endian 64-bit integer pairs to represent
+     either shared or remote memory, schematically structured as
+    - The first two integers (e.g. the first 16 bytes) represent the *total* size (in bytes)
+      of all buffers and the number of buffers in this message (and thus the number of following
+      pairs of ``uint64``)
+    - Each subsequent pair of ``uint64`` values are an address/offset followed the length of
+      that particular buffer.
```

Comment:
Need to fix these markup: http://crossbow.voltrondata.com/pr_docs/41180/format/DissociatedIPC.html#data-stream-sequence